### PR TITLE
Initial iModel.js template

### PR DESCRIPTION
**Pull request by calebmshafer**

### CODE REVIEW

File: packages/cra-template-imodeljs/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# cra-template-typescript
```
Comment (aruniverse) [inline packages/cra-template-imodeljs/README.md:1]: i would change this to imodeljs, as well as update the rest of the readme so we know its an imjs starter

File: packages/cra-template-imodeljs/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+This is the official TypeScript template for [Create React App](https://github.com/facebook/create-react-app).
+
+To use this template, add `--template typescript` when creating a new app.
+
+For example:
+
+```sh
+npx create-react-app my-app --template typescript
```
Comment (aruniverse) [inline packages/cra-template-imodeljs/README.md:10]: should probably mention you can set the scripts version so it uses the bentley fork one

File: packages/cra-template-imodeljs/template.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
+  "package": {
+    "dependencies": {
+      "@bentley/bentleyjs-core": "^2.0.0",
+      "@bentley/extension-client": "^2.0.0",
+      "@bentley/frontend-authorization-client": "^2.0.0",
+      "@bentley/geometry-core": "^2.0.0",
+      "@bentley/imodelhub-client": "^2.0.0",
```
Comment (aruniverse) [inline packages/cra-template-imodeljs/template.json:8]: if we use `^` and dont provide a lock file it'll pull in the latest which is 2.4 right? and if so it'll complain about the new rbac-client dep, so maybe that should be added here as well. and we use ^2.4

File: packages/cra-template-imodeljs/template/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (aruniverse) [inline packages/cra-template-imodeljs/template/README.md:1]: update file to remove references to fb

File: packages/cra-template-imodeljs/template/public/index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (aruniverse) [inline packages/cra-template-imodeljs/template/public/index.html:27]: id change it to iModel.js React App or something along those lines

File: packages/cra-template-imodeljs/template/src/logo.svg [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (aruniverse) [inline packages/cra-template-imodeljs/template/src/logo.svg:1]: havent pulled locally, so dont know what this actually looks like, but want to confirm this svg along with the logo*.png and favicon should be an imodeljs logo

**Comment by calebmshafer** [comment]
Closing as this PR is wildly out of date and with the itwin viewer CRA template.  This is much less needed...

**Pull request closed without merging**